src/gnn_app/gnn_controller.py

```
@@ -1,15 +1,16 @@
+import os
+import pickle
 from typing import List, Optional, Tuple
 
 import matplotlib.pyplot as plt
 import torch
-import os
 
 from gnn_app.datasets import MalwareGraphDataset
 from gnn_app.graph_utils import convert_graphml_xml_to_graphml
```

Ruff (0.8.0)

`gnn_app.graph_utils.convert_graphml_xml_to_graphml` imported but unused

Remove unused import: `gnn_app.graph_utils.convert_graphml_xml_to_graphml`

(F401)